src/lib/API.js

```diff
@@ -1,47 +1,36 @@
-import multer from "multer";
-import { defaultErrorHandler } from '../lib/ErrorHandler';
+import middleware from "./Middleware";
+import { defaultErrorHandler } from "./ErrorHandler";
 
 export const setCORSHeaders = ({ response, url }) => {
   if (url) response.setHeader("Access-Control-Allow-Origin", url);
 };
 
-export const parseFormData = async (req, res) => {
```

Comment:
I'm afraid we cannot move this (for now). `parseFormData` is being imported from `@klaudosl/commons/lib/API` in KlaudSol CMS. Let's retain this, and let's move this when we are doing a major breaking change (v2.0.0)

Reply:
Hello @JohnServo24, `parseFormData` is still not retained, making this change not backwards-compatible.